jobs/update-devserver-static-images.groovy

```diff
@@ -0,0 +1,78 @@
+// The pipeline job to update the dev-server static images in gcloud.
+//
+// When called (which we expect to be every day), this job rebuilds
+// all the images that we use for local dev-server -- one for each
+// first-party service in webapp -- and uploads them to the google
+// container registry.  It then updates a file in webapp saying what
+// git commit corresponds to the latest-pushed images, and checks
+// that into the `automated-commits` branch.
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.kaGit
+//import vars.notify
+//import vars.withTimeout
+
+
+new Setup(steps
+
+).addStringParam(
+   "GIT_BRANCH",
+   """The branch on which to work; we check it out, merge master, and then push
+the updated data-file to it.""",
+   "automated-commits"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to send our status info.",
+   "#infrastructure"
+
+).addCronSchedule(
+   '0 2 * * *'
+
+).apply();
+
+
+def runScript() {
+   // We do our work in the automated-commits branch (first pulling in master).
+   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", params.GIT_BRANCH);
+   kaGit.safeMergeFromMaster("webapp", params.GIT_BRANCH);
+
+   dir("webapp") {
+      sh("timeout 2m git clean -ffd"); // in case the merge modified .gitignore
+      sh("timeout 1m git clean -ffd"); // sometimes we need two runs to clean
+
+      sh("make fix_deps");  // force a remake of all deps all the time
+
+      // Run the script!
+      sh("Make -C dev/server upload-all-static-images");
+   }
+}
+
+
+def publishResults() {
+   dir("webapp") {
+      sh("git add dev/server/tag-of-latest-uploaded-images.txt");
+   }
+   // Check it in!
+   kaGit.safeCommitAndPush(
+      "webapp", ["-m", "Automated update of tag-of-latest-uploaded-images.txt"]);
```

Comment:
Oh! I just realized your other PR is generating this file, not `.env.tags` as I'd previously been using. I kind of prefer the `.env` file since we can just include it and docker knows what to do with it, and it'll still observe what I think most people would expect to be obvious rules around the tags if we want to make it more complicated. That said, this is fine if you prefer this route, it won't be much work for us to just `env BUILD_TAG=$(cat tag-of-latest-uploaded-images.txt)` or whatever on the docker-compose command either.